app/routes/statement.js

```diff
@@ -1,31 +1,52 @@
+const Boom = require('@hapi/boom')
+
 const { getFileStream } = require('../storage')
-const joi = require('joi')
-const boom = require('@hapi/boom')
+const { get, set } = require('../cache')
+
+const streamToBuffer = require('../stream-to-buffer')
+
+const schema = require('./schemas/statement')
 
 module.exports = {
   method: 'GET',
   path: '/statement/{version}/{filename}',

```

Comment:
Cannot test for params version and filename not being present as they are required.

Reason being if we hit a URL where these aren't present it doesn't point to this URL and therefore doesn't validate, hence 404 errors rather than 400.

Schemas are defined as required for potential other needs in the future.